camera_node:
  ros__parameters:
    camera_id: 0
    frame_rate: 30